client/quantum_serverless/core/tracing.py

```diff
@@ -29,7 +29,7 @@
 import os
 from typing import Dict, Optional
 
-from opentelemetry import trace
+from opentelemetry import trace  # pylint: disable=duplicate-code
```

Comment:
One question here @akihikokuroda , why do we need to add this `disable` option? It seems a false positive but just to be sure.

Reply:
It complains about 2 files having the similar set of important statements.